lib/trento/application/integration/discovery/payloads/cluster/sbd_discovery_payload.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Trento.Integration.Discovery.ClusterDiscoveryPayload.Sbd do
+  @moduledoc """
+  SBD field payload
+  """
+  alias Trento.Support.ListHelper
+
+  @required_fields []
+  use Trento.Type
+
+  deftype do
+    embeds_one :config, SBDConfig do
+      field :sbd_delay_start, :string
+      field :sbd_device, :string
+      field :sbd_move_to_root_cgroup, :string
+      field :sbd_pacemaker, :string
+      field :sbd_startmode, :string
+      field :sbd_timeout_action, :string
+      field :sbd_watchdog_dev, :string
+      field :sbd_watchdog_timeout, :string
+    end
+
+    embeds_many :devices, Device do
+      field :device, :string
+      field :status, :string
+    end
+  end
+
+  def changeset(sbd, attrs) do
+    filtered_attrs = transform_nil_lists(attrs)
+
+    sbd
+    |> cast(filtered_attrs, [])
+    |> cast_embed(:config, with: &config_changeset/2)
+    |> cast_embed(:devices, with: &devices_changeset/2)
+    |> validate_required_fields(@required_fields)
+  end
+
+  defp config_changeset(config, attrs) do
+    config
+    |> cast(attrs, [
+      :sbd_delay_start,
+      :sbd_device,
+      :sbd_move_to_root_cgroup,
+      :sbd_pacemaker,
+      :sbd_startmode,
+      :sbd_timeout_action,
+      :sbd_watchdog_dev,
+      :sbd_watchdog_timeout
+    ])
+    |> validate_required_fields([
```

Comment:
I think having these fields in a `@required_config_fields` would look better, but well, just cosmetics hehe